school_lunch/migrations/17.0.1.0.5/end-10-migrate.py

```diff
@@ -0,0 +1,6 @@
+from odoo.upgrade import util
+
+
+def migrate(cr, _):
+    env = util.env(cr)
+    env["school_lunch.order"].search([("date_end_gantt", "=", False)])._compute_date_end_gantt()
```

Comment:
btw you can use `util.recompute_fields()`
```
recomputed_records = env["school_lunch.order"].search([("date_end_gantt", "=", False)])
util.recompute_fields(cr, "school_lunch.order", ["date_end_gantt"], ids=recomputed_records.ids)
```

Reply:
I haven't ever used the `recompute_fields` method. I didn't think about it... Thanks, I will commit your suggestion.